infra/eligibility_screener/environments/test/.terraform/modules/modules.json

```diff
@@ -0,0 +1 @@
+{"Modules":[{"Key":"","Source":"","Dir":"."},{"Key":"template","Source":"../../template","Dir":"../../template"},{"Key":"template.constants","Source":"../constants","Dir":"../../constants"}]}
```

Comment:
you can probably delete this file if you're having trouble removing it from git tracking 😛